src/documents/Document.js

```diff
@@ -1,10 +1,12 @@
 // @flow
 
 class DummyEngine implements DocumentGeneratorEngine {
-  process = async (data: mixed, pathToTemplate: string) => {
-    console.warn("calling process without proper engine", data, pathToTemplate);
+  process = async (data: mixed, templates: Array<string>) => {
+    console.warn("calling process without proper engine", data, templates);
 
-    return Promise.resolve([null, ""]);
+    const buffer = new Buffer(256);
+
+    return Promise.resolve([[buffer, ""]]);
```

Comment:
By returning null you can probably be explicit about the fact that we didn't do anything with the templates and the data, don't you think?

Reply:
This is to comply with the interface, null here will be an empty buffer. We can make it explicity null if you think that's better, but I'm not sure how to make that in Flow, suggestions welcomed @duranmla

Reply:
We can add a question mark  @orlando to one side of the returning type like:
`: Promise<Array<[?Buffer, string]>>`